ci/prepare.sh

```diff
@@ -65,14 +65,6 @@ sudo apt-get remove -y ansible
 sudo pip install ansible==2.0.2.0
 sudo pip install --upgrade setuptools
 
-# Prepate the docker-ovn-hosts file
-LOCALIP=$(ip addr show dev eth0 | grep 'inet ' | cut -d " " -f 6 | cut -d "/" -f 1)
-if [ "$LOCALIP" == "" ] ; then
-    # Try bond0
-    LOCALIP=$(ip addr show dev bond0 | grep 'inet ' | cut -d " " -f 6 | cut -d "/" -f 1)
-fi
-cat ansible/docker-ovn-hosts-example | sed -e "s/REPLACE_IP/$LOCALIP/g" > ansible/docker-ovn-hosts
```

Comment:
@huikang: the lines removed are actually duplicated down below in that same file.
With that, they are just not needed.
Note that line 74 is moved, not removed, to the lines below.